### ISSUE TITLE
macOS Sierra compatibility?

### ISSUE THREAD

**Issue by victorjkhalaf** [Commander147/CMStorm#2]
I tried to run the app under Sierra, and it failed to load up in the menu bar. Accessibility was granted, and I can't see any indication that something is failing. 

Wondering if you tested it out?

**Comment by jafework** [comment]
@victorminator I haven't tested this against Sierra yet, but were you able to get it running?

**Comment by victorjkhalaf** [comment]
Alas no, I ended up trying a different solution that worked. I'm sorry I couldn't be of further help. I tried to delete this post so it didn't appear and have no benefit to you or users but the most I could do was try and close it.
